### ISSUE TITLE
Dismissing a HUD interferes with the VoiceOver focus

### ISSUE THREAD

**Issue by lapfelix** [SVProgressHUD/SVProgressHUD#771]
`UIAccessibilityPostNotification(UIAccessibilityScreenChangedNotification, nil)`[ is called whenever a HUD is dismissed,](https://github.com/SVProgressHUD/SVProgressHUD/blob/master/SVProgressHUD/SVProgressHUD.m#L1065) which plays the screen changed VoiceOver sound but also moves the focus to the first accessible item in the screen.

I think that if this line has to be called, it should be done in the optional completion handler because it does more than just play the screen changed sound.

**Comment by honkmaster** [comment]
I checked the [documentation](https://developer.apple.com/reference/uikit/uiaccessibilityscreenchangednotification). It says:

"Posted by an application when a new view **appears** that makes up a major portion of the screen."

Thus, I think it would be O.K. to remove the line from `dismissWithDelay:` as no new view appears.

**Comment by lapfelix** [comment]
Yes that's exactly what I intended to do. I'll make a pull request.

**Comment by honkmaster** [comment]
Great. Thanks for the contribution.
